### PR TITLE
Delete priority functionality

### PR DESCRIPTION
**Pull request by Sahil25061999**
added delete page and priority filter 

Files to check:
- NotesInputModal.jsx
- Filter.jsx
- pages/ delete
- context / delete-list-context

Login email: adarshbalika@gmail.com
password: adarshBalika

### CODE REVIEW

File: src/api-call/getArchive.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+
+export const getArchive = async (token) => {
+  try {
+    const response = await axios.get('/api/archives', { headers: token });
+    if (response.status === 200 || response.status === 201) {
```
Comment (yaswanthmyneni) [inline src/api-call/getArchive.js:6]: change: here status === 200 is enough

File: src/component/filter/Filter.css [hunk 2]
```diff
@@ -97,3 +98,10 @@ option {
   padding: 0.5rem;
   box-shadow: 4px 8px 16px rgba(58, 58, 58, 0.112);
```
Comment (yaswanthmyneni) [inline src/component/filter/Filter.css:99]: changes: css variables for colors

File: src/component/notesInputModal/NotesInputModal.jsx [hunk 3]
```diff
@@ -60,6 +61,21 @@ export const NotesInputModal = () => {
...
+      case 'PRIORITY_MEDIUM':
+        console.log('med called');
+        return noteDispatch({ type: 'PRIORITY_MEDIUM' });
+
+      case 'PRIORITY_HIGH':
+        return noteDispatch({ type: 'PRIORITY_HIGH' });
+      case 'PRIORITY_LOW':
+      default:
+        console.log('called');
+        return noteDispatch({ type: 'PRIORITY_LOW' });
```
Comment (yaswanthmyneni) [inline src/component/notesInputModal/NotesInputModal.jsx:74]: changes: remove console.log when pushing code to github

**Review by yaswanthmyneni — APPROVED** [review]
you can merge after changes.

**Review by nikhiljugale007 — APPROVED** [review]
Did not find any issues apart from mentioned above. 
Ui is very clean.  
Good to merge.

**Pull request merged**